api.js

```diff
@@ -177,24 +187,30 @@ Api.prototype.run = function (files) {
 			var tests = files.map(self._runFile);
 
 			// receive test count from all files and then run the tests
+			var runOnlyExclusive = false;
 			var statsCount = 0;
 
 			return new Promise(function (resolve) {
 				tests.forEach(function (test) {
 					var counted = false;
 
-					function tryRun() {
+					function tryRun(stats) {
 						if (counted) {
 							return;
 						}
 
+						runOnlyExclusive = runOnlyExclusive || stats.hasExclusive;
```

Comment:
To keep to test execution concerns contained in `tryRun()` and separate from the bookkeeping concerns in `_handleStats()`. But it's not a big hangup for me. I'll go ahead and update `tryRun()` to use `self.hasExclusive`.